python/README.md

```diff
@@ -0,0 +1,53 @@
+---
+
+## Introduction
+
+  * **What is Robyn?**: Robyn is an experimental, semi-automated and open-sourced Marketing Mix Modeling (MMM) package from Meta Marketing Science. It uses various machine learning techniques (Ridge regression, multi-objective evolutionary algorithm for hyperparameter optimization, time-series decomposition for trend & season, gradient-based optimization for budget allocation, clustering, etc.) to define media channel efficiency and effectivity, explore adstock rates and saturation curves. It's built for granular datasets with many independent variables and therefore especially suitable for digital and direct response advertisers with rich data sources. 
+  
+  * **Why are we doing this?**: MMM used to be a resource-intensive technique that was only affordable for "big players". As the privacy needs of the measurement landscape evolve, there's a clear trend of increasing demand for modern MMM as a privacy-safe solution. At Meta Marketing Science, our mission is to help all businesses grow by transforming marketing practices grounded in data and science. It's highly aligned with our mission to democratizing MMM and making it accessible for advertisers of all sizes. With Project Robyn, we want to contribute to the measurement landscape, inspire the industry and build a community for exchange and innovation around the future of MMM and Marketing Science in general.
+  
+## Quick start for Python (Beta)
+
+  The Python version of Robyn was developed by utilizing Code Llama's capabilities to port code from R to Python. As is common with any AI-based solutions, there may be potential challenges in translating code from one language to another.
+  In this case, we anticipate that there could be some issues in the translation from R to Python. However, we believe in the power of community collaboration and open-source contribution. Therefore, we are opening this project to the community to participate and contribute.
+  Together, we can address and resolve any issues that may arise, enhancing the functionality and efficiency of the Python version of Robyn. We look forward to your contributions and to the continuous improvement of this project. 
+
+**1. Installing the package**
+  
+  * Install Robyn latest package version:
+```{r}
+## Pypi
+pip3 install robynpy
+
+## DEV VERSION
+# if you are pulling source from github, install dependencies using requirements.txt
+pip3 install -r requirements.txt
+```
+  
+**2. Getting started**
+
+  * Use this [demo.py](https://github.com/facebookexperimental/Robyn/tree/main/python/src/scripts/demo.py) script as step-by-step guide that is
```

Comment:
demo.py does not exist at this time. nit: we might want to flag that this would be included in a future PR?

Reply:
demo.py will be replaced by set of tutorials. Let me update this.